Mathlib/Topology/Order/ScottTopology.lean

```diff
@@ -111,11 +111,11 @@ lemma ScottHausdorff.Lower_IsOpen {s : Set α} (h : IsLowerSet s) : ScottHausdor
 The Scott topology is defined as the join of the topology of upper sets and the Scott Hausdorff
 topology.
 -/
-def Scott : TopologicalSpace α := upperSet α ⊔ ScottHausdorff
+def Scott : TopologicalSpace α := upperSet α ⊔ scottHausdorff
```

Comment:
```suggestion
def scott : TopologicalSpace α := upperSet α ⊔ scottHausdorff
```

Reply:
Done, thanks.

Reply:
Seemingly, you didn't actually push the change! It's still shown `Scott` on the diff.

Reply:
Sorry! Hopefully I have corrected this now.